examples/apps/aml/STATEFUL.md

```diff
@@ -0,0 +1,11 @@
+
+## Changes to make tcf stateful
```

Comment:
@EugeneYYY has some questions about this.

Reply:
We thought that it will be easier for everyone if at this moment we only add the example and after that we discuss how to make Avalon stateful

Reply:
Could you remove this file from this PR